skfp/other/util_transformers.py

```diff
@@ -0,0 +1,91 @@
+from typing import Union
+
+import numpy as np
+import pandas as pd
+from rdkit.Chem import MolFromSmiles, MolToSmiles
+from rdkit.Chem.rdchem import Mol
+
+from skfp.fingerprints.base import FingerprintTransformer
+
+
+class MolFromSmilesTransformer(FingerprintTransformer):
+    def __init__(
+        self,
+        sanitize: bool = True,
+        replacements: dict = {},
+        random_state: int = 0,
+        n_jobs: int = None,
+        verbose: int = 0,
+        **kwargs
+    ):
+        super().__init__(
+            n_jobs=n_jobs,
+            sparse=False,
+            count=False,
+            verbose=verbose,
+            random_state=random_state,
+        )
+        self.sanitize = sanitize
+        self.replacements = replacements
+        self.fingerprint_transformer = False
+
+    def _calculate_fingerprint(
+        self, X: Union[pd.DataFrame, list[str]]
```

Comment:
This won't work for `pd.DataFrame`, nor for `pd.Series`, as it is. I would leave just `list[str]`